packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -129,13 +129,19 @@ export class RedirectsMiddleware extends MiddlewareBase {
           .replace(regexParser(existsRedirect.pattern), existsRedirect.target)
           .replace(/^\/\//, '/')
           .split('?');
-        url.pathname = target[0];
+
+        url.pathname = `${target[0]}`;
+
         if (target[1]) {
           const newParams = new URLSearchParams(target[1]);
-          for (const [key, val] of newParams.entries()) {
-            url.searchParams.append(key, val);
-          }
+          url.search = `?${newParams.toString()}`;
```

Comment:
Is it fine to change URL property directly? previously it was url.searchParams.append

Reply:
Yes, there uses url.search in below